xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -756,6 +761,19 @@ void CMusicInfoTag::Serialize(CVariant& value) const
   value["musicbrainzalbumartistid"] = m_musicBrainzAlbumArtistID; 
   value["musicbrainztrmid"] = m_strMusicBrainzTRMID;
   value["comment"] = m_strComment;
+  value["contributors"] = CVariant(CVariant::VariantTypeArray);
+  for (unsigned int i = 0; i < m_musicRoles.size(); ++i)
```

Comment:
use for (const auto& role : m_musicRoles and then access as role.GetArtist(); instead of m_musicRoles{i].GetArtist();